### ISSUE TITLE
Symlinks?

### ISSUE THREAD

**Issue by wilsonmar** [jnichols3/terraform-envs#2]
Can you include symlinks in the repo here? 
Either as static links or in shell file which creates them.
Are symlinks different in MacOS vs. different flavors of Linux?
The reference to "symlinks" in the README - does that mean those stored at /Library/Program on MacOS but point to the physical location elsewhere (such as /Volumes/Program)? 
See https://towardsdatascience.com/reflinks-vs-symlinks-vs-hard-links-and-how-they-can-help-machine-learning-projects-b77b89cdbab1

**Comment by jnichols3** [comment]
> Can you include symlinks in the repo here?

Yes, symlinks are included. For example, this project level symlink is a relative reference to the correct file in the base directory:
https://github.com/jnichols3/terraform-envs/blob/main/projects/ec2/prod/aws-vars.tf

All of the symlinks are platform independant and are all relative to the root of the repository and should be portable.
